src/dsp_tools/models/custom_warnings.py

```diff
@@ -19,7 +32,16 @@ class DspToolsUserWarning(Warning):
     @classmethod
     def showwarning(cls, message: str) -> None:
         """Print the warning, without context"""
-        cprint(f"WARNING: {message}", color="red", attrs=["bold"])
+        print(BOLD_RED + f"WARNING: {message}" + RESET_TO_DEFAULT)
+
+
+class DspToolsUserInfo(Warning):
+    """Class for general user-facing warnings"""
```

Comment:
Identical docstring as above, perhaps you want to formulate one specific for this class?